pages/doc.js

```diff
@@ -1,21 +1,20 @@
 /* global docsearch:readonly */
 
-import React, { Component } from 'react'
+import React, { useCallback, useState, useEffect } from 'react'
+import PropTypes from 'prop-types'
+import Error from 'next/error'
+import Router from 'next/router'
 // components
 import Page from '../src/Page'
 import { HeadInjector } from '../src/Documentation/HeadInjector'
 import Hamburger from '../src/Hamburger'
 import SearchForm from '../src/SearchForm'
 import SidebarMenu from '../src/Documentation/SidebarMenu/SidebarMenu'
-import Loader from '../src/Loader/Loader'
```

Comment:
if we remove them here, let's remove corresponding components?

Reply:
Removed them for now

Reply:
👍